test/draw/gpu/GpuImageRotationTest.ooc

```diff
@@ -45,6 +45,7 @@ GpuImageRotationTest: class extends Fixture {
 		this add("GPU rotation small X (BGRA)", func {
 			correctImage := RasterBgra open("test/draw/gpu/correct/rotation_small_bgra_X.png")
 			gpuImage := gpuContext createBgra(sourceImage size)
+			gpuImage canvas pen = Pen new(ColorBgra new())
```

Comment:
How did this make any difference? I thought the pen was already defaulted to `ColorBgra new()` in the Canvas?